### DISCUSSION TITLE
Export to HTML - color instead underline

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1636] [Q&A] [answered]
When I export a page of Zim's notebook to HTML, I see in the browser that the underline of the text turns into the color of its background.

As I understand it, the problem is on the HTML side ?

Is it possible (and how) to change the export template to make this problem go away?

**Comment by introt** [comment]
Change the `u` (underline) to `mark` (highlight) in this line: 

https://github.com/zim-desktop-wiki/zim-desktop-wiki/blob/02e295f22991deddaf85e5ea5ebdcf5aadbbea87/zim/formats/html.py#L59

so that the line looks like this: `MARK: ('<mark>', '</mark>'),` and then re-export.

**Reply by introt** [reply]
The website refers to this formatting as `highlight`, while the menu option is named `Mark` and the UI symbol is for underline. We could totally have _both_, though that'd be a breaking change and a half.

**Comment by Nick222** [comment]
I would not like to change the program code - especially since I have to do it after each update.

**Comment by introt** [comment, marked as answer]
Sorry, I think I misunderstood you the first time. Here's how to change highlight to underline:

1. Open the template
2. Remove these lines:
```
		u          { text-decoration: none;
					 background-color: yellow   }
```

### Before:
![image](https://user-images.githubusercontent.com/26069787/135651988-62f04b72-0082-429d-a9de-dcbe2bbb3cf0.png)

### After:
![image](https://user-images.githubusercontent.com/26069787/135654179-005a8c8b-8045-4422-bdd5-30bea77f4204.png)

**Reply by introt** [reply]
No, sorry, I don't think you meant this either. Could you post screenshots of what's wrong as well as what it should look like?

**Comment by Nick222** [comment]
Very good!

I edit template as:

`u          { text-decoration: underline }`

and everything began to look like in your pictures...

This is exactly what I wanted !

Thank you ! :)
